### ISSUE TITLE
Isaac: Side special Blockable

### ISSUE THREAD

**Issue by ghost** [TheOriginalSINe/Project-M-CC#163]
Isaac's side special doesn't push the opponent when they're blocking.

**Comment by ghost** [comment]
This may be intended.

**Comment by bb010g** [comment]
I wouldn't think so. It can already be destroyed with a light attack, and only needing to shield makes it exceedingly hard to space with Push.

**Comment by Eltrion** [comment]
It really just needs to be better. If it really is supposed to be blockable, then it should also be bigger, and able to take more damage, and probably faster as well.

**Comment by bb010g** [comment]
I think it already was faster in the [leak video](https://www.youtube.com/watch?v=HvSxLZKvtnU), although it would probably be good to try more damageable.

**Comment by Nekudot** [comment]
It's just an unfinished move. The size and speed are already bad enough, but it can be blocked and destroyed with a single hit? It's almost useless. I agree that speed and durability need to be increased if it can be blocked and remains at that size.

**Comment by somealternateaccount** [comment]
Putting on the Isaac completion milestone for now since this is potentially a balance-affecting change. Of additional note is how easy it is to powershield Psy Move, since it inflicts no damage or hitstun and thus is always inside an opponent's powershield bubble.

**Comment by BlackDrMario** [comment]
You can even power shield the hand back to him. Honestly I think it's fine. If they come out of block they get pushed back. If they don't then it's a potential grab if they don't spot dodge.

The size I think works fine, I think it looks bigger than the model suggests. As for speed, any faster could make it a tad OP, imo. The speed is just right that you can catch up to it and act accordingly with it present.

tldr; it's not useless at all. Especially if someone tries to wavedash from ledge.

**Comment by somealternateaccount** [comment]
I feel like maybe it shouldn't be reflectable? There's already the simple counterplay of hitting it at all, and powershield working so easily on it feels a bit strange and weak. We'd likely need a more fleshed-out Isaac before we could judge the balance of this move, however.
